backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/TransitionUsageImpl.java

```diff
@@ -163,15 +161,12 @@ public ActionUsage getSource() {
      * @generated NOT
      */
     public ActionUsage basicGetSource() {
-        ActionUsage result = null;
-        Optional<Membership> membership = this.getOwnedMembership().stream().findFirst();
-        if (membership.isPresent()) {
-            Element memberElement = membership.get().getMemberElement();
-            if (memberElement instanceof ActionUsage actionUsage) {
-                result = actionUsage;
-            }
+        // See specification constraint deriveTransitionUsageSource in TransitionUsage
+        Feature sourceFeature = this.sourceFeature();
```

Comment:
Nope, this is not what is described in the specification.
I will update it to conform to the sepcification.